### ISSUE TITLE
Missing useful tags

### ISSUE THREAD

**Issue by SalahAdDin** [takeflight/wagtail-metadata#32]
I wants ask for next tags:

```
<meta name="twitter:site" content="@">
<meta name="twitter:creator" content="@">
<meta property="fb:app_id" content="995117193836517" />
<meta property="og:type" content="website" />
```

Have you plans for do it?

Thanks you!!!!

**Comment by SalahAdDin** [comment]
@SableWalnut 
Man, i was seeing this package: https://github.com/bashu/wagtail-metadata-mixin/blob/develop/wagtailmetadata/models.py

Can you see it? it applies many other tags, unfortunately this package isn't currently in development.

**Comment by SableWalnut** [comment]
Closed with #33.

**Comment by SalahAdDin** [comment]
@SableWalnut Yes, but in the other package there are many other very useful.
